examples/tokenvm/cmd/token-cli/cmd/action.go

```diff
@@ -117,12 +119,39 @@ var createAssetCmd = &cobra.Command{
 			return err
 		}
 
+		promptSupply := promptui.Prompt{
+			Label: "Initial Supply (empty or '0' for 'none')",
+			Validate: func(input string) error {
+				if input == "" {
+					return nil
+				}
+				_, err := strconv.ParseUint(input, 10, 64)
+				if err != nil {
+					return err
+				}
+				return nil
+			},
+		}
+		supplyText, err := promptSupply.Run()
+		if err != nil {
+			return err
+		}
+
 		// Confirm action
 		cont, err := promptContinue()
 		if !cont || err != nil {
 			return err
 		}
 
+		var supply uint64 = 0
+
+		if supplyText != "" {
```

Comment:
Won't be necessary after migrating to `promptInt`

Reply:
done